modules/basics/src/main/java/com/opengamma/strata/basics/location/Country.java

```diff
@@ -256,11 +259,12 @@ public final class Country
    * <p>
    * This contains all the countries that have been defined at the point
    * that the method is called.
-   * 
+   *
    * @return an immutable set containing all registered countries
    */
   public static Set<Country> getAvailableCountries() {
-    return ImmutableSet.copyOf(CACHE.values());
+    COUNTRY_CODES.get();
+    return ImmutableSortedSet.copyOf(CACHE.values());
```

Comment:
I'm pretty sure this is enough because of the ConcurrentSkipListMap. Worth checking though

```suggestion
    return CACHE.values();
```

Reply:
`CACHE.values()` returns a`Collection` so I used the `ImmutableSortedList` to avoid updating the return type.